frontend/institution/managementMembersController.js

```diff
@@ -271,6 +259,42 @@
             (field.email === empty_email.email) ? removeField(field) : addField();
         };
 
+        manageMemberCtrl.removePendingAndMembersEmails = function removePendingAndMembersEmails(emails) {
+            var requestedEmails = manageMemberCtrl.requests
+                .map(request => {
+                    return request.institutional_email;
+                });
+
+            var invitedEmails = manageMemberCtrl.sent_invitations
+                .map(invite => {
+                    return invite.invitee;
+                });
+
+            var membersEmails = [].concat.apply([], manageMemberCtrl.members
```

Comment:
Porque cada email de um membro é um array, então iria ficar um array de arrays. O que não é viável para o caso.

Reply:
**var membersEmails = manageMemberCtrl.members.reduce((emails, member) => [...emails, ...member.email], []);**

This way it is simpler to understand. Since it is to reduce the emails lists of users in one, you can use the reduce. See if it makes more sense, otherwise disregard.